jaws-journey-deploy/orb.yml

```diff
@@ -237,6 +246,61 @@ commands:
             npx -p @semantic-release/changelog -p @semantic-release/git -p semantic-release semantic-release --debug=true --tag-format=\${version} --branches='master' --plugins='@semantic-release/commit-analyzer' '@semantic-release/release-notes-generator' '@semantic-release/changelog' '@semantic-release/git' '@semantic-release/github' --repositoryUrl="https://github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}.git"
           name: Run semantic versioning
 
+
+  tf:
+    parameters:
+      <<: *tf-common-parameters
+      command:
+        type: "enum"
+        enum: [ "plan", "apply" ]
+        default: "plan"
+      install_java:
+        type: boolean
+        default: false
+    description: tf << parameters.command >> << parameters.environment >>
+    steps:
+      - when:
+          condition: << parameters.attach_workspace >>
+          steps:
+            - attach_workspace:
+                at: ~/working
+      - unless:
+          condition: << parameters.attach_workspace >>
+          steps:
+            - checkout
+      - when:
+          condition: << parameters.install_java >>
```

Comment:
cant you just have this as a separate step?  Then the tf command is generic and you just have an additional step at 586?